### PR TITLE
Bugfix: simplify getSfHost

### PR DESCRIPTION
**Pull request by mehdisfdc**
## Describe your changes

This is to fix a regression on .mcas domains (maybe after [PR#141](https://github.com/tprouvot/Salesforce-Inspector-reloaded/pull/141/)?)
I simplified the `getSfHost` logic: now it simply returns the current domain.
**I'm not sure why the previous logic was so complex**, since the extension should not show on non-Salesforce domains based on its manifest. But maybe I missed the point, and broke something? @tprouvot @victorgz your feedback is more than welcome.

## Checklist before requesting a review
- [X] I have read and understand the [Contributions section](https://github.com/tprouvot/Salesforce-Inspector-reloaded#contributions)
- [X] Target branch is releaseCandidate and not master
- [X] I have performed a self-review of my code
- [X] I ran the [unit tests](https://github.com/tprouvot/Salesforce-Inspector-reloaded#unit-tests) and my PR does not break any tests
- [ ] I documented the changes I've made on the [CHANGES.md](https://github.com/tprouvot/Salesforce-Inspector-reloaded/blob/master/CHANGES.md) and followed actual conventions
- [ ] I added a new section on [how-to.md](https://github.com/tprouvot/Salesforce-Inspector-reloaded/blob/master/docs/how-to.md) (optional)

### CODE REVIEW

**Comment by mehdisfdc** [comment]
@yaacov9 Since you are the author of PR#141, your feedback on my proposed simplified logic to getHost is also welcome :-)

File: addon/background.js [hunk 1]
```diff
@@ -13,23 +13,7 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
...
-          let sessionCookie = cookies.find(c => c.value.startsWith(orgId + "!"));
-          if (sessionCookie) {
-            sendResponse(sessionCookie.domain);
-          } else if (orderedDomains[orderedDomains.length] === currentDomain){
-            sendResponse(currentDomain);
-          }
-        });
-      });
+      sendResponse(currentDomain);
     });
```
Comment (victorgz) [inline addon/background.js:17]:
```suggestion

```

**Comment by victorgz** [comment]
Hi @mehdisfdc, I've just added a suggestion on how to clean even more the code. However, I'm still trying to figure out why that code was there and what would be the impact if we change it... I'm not really sure yet 😅 I'll let you know if I get any insight

**Comment by mehdisfdc** [comment]
@victorgz @tprouvot Given the old comment in the code I suppose it was for Visualforce use cases, but I could not find a test case that worked with the old code and does not work anymore...

**Comment by yaacov9** [comment]
I don't think your code will work in every case, but I'm not sure, and my PR just add some domains so I don't think it's the cause of .mcas domains.
Also, with my short experience in this project I think the old way is the right way to check the cookie
For example in Hyperforce hosted on Alibaba cloud, your domain is following these patterns: 
- https://*.sfcrmapps.cn/* for Salesforce Lightning
- https://*.sfcrmproducts.cn/* for Salesforce Classic
But the session cookie will always be saved from the domain https://*.sfcrmproducts.cn/*, so you can't based only on the actual domain you are
but I could be wrong

**Comment by mehdisfdc** [comment]
Thank you @yaacov9 for looking into it.
I'm not familiar with the behaviour of Hyperforce on Alibaba cloud, so yes maybe there is something I missed.

If you have access to a Salesforce instance hosted on on Alibaba cloud,, would it be possible for you to do a quick tests with the code of this PR? 
If you confirm it breaks something, I will revert to the old logic and look for another solution to restore compatibility with .mcas.ms domains.

**Comment by tprouvot** [comment]
@mehdisfdc it looks like there is a regression for Alibaba Cloud orgs

image

**Comment by mehdisfdc** [comment]
Thank you for the confirmation @tprouvot !

Then I'll close this PR, and reopen one with the previous code + a fix for the mcas use-case.

**Pull request closed without merging**